test/Dockerfile.py3

```diff
@@ -1,4 +1,4 @@
-FROM metabrainz/python:3.5
+FROM metabrainz/python:3.6
```

Comment:
Does this change have possible downstream ramifications that need to be considered?

Reply:
I guess this is the docker file for running tests on travis? As long as LB, CB, MeB web are running on python 3.6 this is not an issue.

Reply:
Yep, this is only for tests and all of LB, CB and MeB run on the `metabrainz/python3.6` image.

Reply:
ok, great.